src/providers/bitcoin/BitcoinSwapProvider.js

```diff
@@ -52,7 +50,16 @@ export default class BitcoinSwapProvider extends Provider {
   }
 
   async claimSwap (initiationTxHash, recipientAddress, refundAddress, secret, expiration) {
-    const secretHash = sha256(secret)
+    return this._redeemSwap(initiationTxHash, recipientAddress, refundAddress, secret, expiration, true)
+  }
+
+  async refundSwap (initiationTxHash, recipientAddress, refundAddress, secretHash, expiration) {
+    return this._redeemSwap(initiationTxHash, recipientAddress, refundAddress, secretHash, expiration, false)
+  }
+
+  async _redeemSwap (initiationTxHash, recipientAddress, refundAddress, secretParam, expiration, isClaim) {
+    const secretHash = isClaim ? sha256(secretParam) : secretParam
+    const lockTime = isClaim ? 0 : expiration + 100
```

Comment:
why the +100?

Reply:
Because locktime must be greater than locktime set in the redeem script in order to pass.

Reply:
I chose 100 arbitrarily (100 seconds seems reasonable).